source/includes/stackpath/_dns_records.md

```diff
@@ -0,0 +1,123 @@
+## DNS Records
+
+Create and manage your DNS records associated to your zones.
+
+<!-------------------- LIST DNS RECORDS -------------------->
+
+### List DNS Records
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/dnsrecords?zoneId=337c7c26-31f4-4b2e-83dc-126exxxxxxWr"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": "4eea2e59-8d06-4f37-8fd1-035e3314dxxD",
+      "stackId": "7fbea2c0-17d0-41df-b0b1-4d5c95234566",
+      "zoneId": "337c7c26-31f4-4b2e-83dc-126exxxxxxWr",
+      "name": "test1",
+      "type": "A",
+      "classCode": "IN",
+      "ttl": 21600,
+      "data": "127.0.0.0",
+      "weight": 1,
+      "created": "2021-07-28T17:14:07.928210Z",
+      "updated": "2021-07-28T17:14:07.928210Z"
+    },
+    {
+      "id": "0a18b323-72df-447a-af81-08a2ea62re45",
+      "stackId": "7fbea2c0-17d0-41df-b0b1-4d5c95234566",
+      "zoneId": "337c7c26-31f4-4b2e-83dc-126exxxxxxWr",
+      "name": "test2",
+      "type": "TXT",
+      "classCode": "IN",
+      "ttl": 300,
+      "data": "123 312 12",
+      "weight": 1,
+      "created": "2021-07-28T17:14:25.229376Z",
+      "updated": "2021-07-28T17:14:25.229376Z"
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords?zoneId=:zoneId</code>
+
+Retrieve a list of all DNS records in a given DNS zone in an [environment](#administration-environments) .
+
+Query Params | &nbsp;
+---- | -----------
+`zoneId`<br/>*UUID* | The ID of the zone for which we list the records. This parameter is required.
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | A record's unique identifier. 
+`stackId`<br/>*string* | The ID of the stack that a record belongs to.
+`zoneId`<br/>*string* | The ID of zone that the record belongs to.
+`name`<br/>*string* | The name of the record. Use the value "@" to denote current root domain name.
+`type`<br/>*string* | A zone record's type. Zone record types describe the zone record's behavior.
+`classCode`<br/>*string* | A zone record's class code. This is typically "IN" for Internet related resource records.
+`ttl` <br/>*int* | A zone record's time to live. A record's TTL is the number of seconds that the record should be cached by DNS resolvers. Use lower TTL values if you expect zone records to change often. Use higher TTL values for records that won't change to prevent extra DNS lookups by clients.
+`data`<br/>*string* | A zone record's value. MX record data follows the format `<priority>` `<value>`, without <>s.
```

Comment:
Not sure if we should sepcifc the MX records. If we do, we sould need to specify for each type

Reply:
ab69727fae9aab6231fae910c34d80912956964b Took it from the Stackpath doc, it would be quite long to have all the MX values so I removed it.